functions/utils.js

```diff
@@ -22,7 +22,7 @@ export async function pushToGitHub(content, filename, nbUid) {
   })
   try {
     const res = await octokit.request(
-      'PUT /repos/{owner}/{repo}/contents/{path}',
+      'GET /repos/{owner}/{repo}/contents/{path}',
```

Comment:
do you still need message and content for get request here ?

Reply:
Most probably no, I was checking the netlify logs to check what exactly is the download url being returned...